Formula/macvim.rb

```diff
@@ -2,9 +2,9 @@
 class Macvim < Formula
   desc "GUI for vim, made for macOS"
   homepage "https://github.com/macvim-dev/macvim"
-  url "https://github.com/macvim-dev/macvim/archive/refs/tags/release-174.tar.gz"
+  url "https://github.com/macvim-dev/macvim/archive/refs/tags/release-176.tar.gz"
   version "9.0.472"
```

Comment:
I think this needs to be updated to "9.0.1276" (see https://github.com/macvim-dev/macvim/releases/tag/release-176)

Reply:
Good catch. I updated the version...